content/news/2015/12/2015-12-28-ocsits-2015-customer-survey-what-we-learned.md

```diff
@@ -12,7 +12,7 @@ topics:
   - customer-service
   - dap
   - digital-analytics-program
-  - ecpic
+  - budgeting-and-performance
```

Comment:
Budgeting and performance is the one that ecpic was merged into. If you take a look at line 418 in the spreadsheet it shows that the merge of the 2 topics